packages/package_info/example/lib/main.dart

```diff
@@ -72,7 +92,11 @@ class _MyHomePageState extends State<MyHomePage> {
       appBar: new AppBar(
         title: const Text('Plugin example app'),
       ),
-      body: new Center(child: new Text('App version is: $_version\n')),
+      body: new Column(mainAxisAlignment: MainAxisAlignment.center, children: <Widget>[
```

Comment:
Not formatted correctly ([Travis build failure](https://travis-ci.org/flutter/plugins/jobs/327547685)): add line break after `MainAxisAlignment.center,`.